2020/day-1-report-repair/index.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { findSumParts } from './'
+
+describe('Day 1: Report Repair', () => {
+  it('Finds a number pair whose sum equals 2020, then multiplies those two numbers', () => {
+    const numbers = [1721, 979, 366, 299, 675, 1456]
+    const [a, b] = findSumParts(2020, numbers)
+    const expected = 514579
```

Comment:
Personally I would prefer to see this inline as it would be easier to read in line 9.

Reply:
Yeah I've never quite worked out which I prefer. I think extracting like this makes more sense when the expected value is a bit more complex, e.g. a large object. I think I don't really have a preference either way so will go with your suggestion